tools/crm_mon.c

```diff
@@ -1472,22 +1481,24 @@ main(int argc, char **argv)
     }
 
     do {
-        out->info(out,"Waiting until cluster is available on this node ...");
-
+#if CURSES_ENABLED
+        if (output_format == mon_output_console) {
+            clear();
+            refresh();
+        }
+#endif
```

Comment:
Moving this block to the top of the do-while loop is an accidental leftover. It gets undone in the final commit. I'll undo this change before we merge anything.